### DISCUSSION TITLE
Welcome to raspberry-gateway Discussions!

### DISCUSSION THREAD

**Discussion by d3vilh** [d3vilh/raspberry-gateway#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fellowape** [comment]
It is good to have direct links from webinstall to documentation.

**Comment by RobotsAreCrazy** [comment]
Thankyou for implementing all my suggestions, pretty cool, are you sure there's no way how you can add custom privatedns etc for using control-d etc?
